packages/mui-system/src/createStyled.js

```diff
@@ -245,3 +234,41 @@ export default function createStyled(input = {}) {
     return muiStyledResolver;
   };
 }
+
+export function applyStyled(props, componentName, overridesResolver) {
+  const styles = applyThemeOverrides(props, componentName, overridesResolver);
+
+  // NOTE: Later on, we might want to apply other MUI features:
```

Comment:
Are the theme overrides at this point the only thing MUI X needs? If users of MUI X used variants, will it still work?

Reply:
Good point, it's extremely unlikely but not impossible, I could enable them right now.